### DISCUSSION TITLE
Plug-ins Extras

### DISCUSSION THREAD

**Discussion by Almighty188** [winsiderss/systeminformer#1518] [Questions / Answers]
Will there be Plug-ins Extras for System Informer like there was for Process Hacker?  If so, it would help if there were periodic builds of the plug-ins extras available unless there was a way to build them on github.

**Comment by Biswa96** [comment]
Some plugins were imported in SystemInformer from plugin-extra repository. If you can provide which plugins you want and its use case, then it may help developers to prioritize those.

**Reply by Almighty188** [reply]
Which ones were imported? Normally, I just use the ones that were compiled by VictorVG but is seems he hasn't provided anything new since November 2021.  Don't really want to specify which ones I want since it's better to have the tools when one needs it.

**Reply by Biswa96** [reply]
Some of those listed here https://github.com/winsiderss/systeminformer/pulls?q=is%3Apr+migrate+is%3Aclosed+extendedtools

**Reply by Almighty188** [reply]
Seems like the only ones that doesn't exist in the list above when compared to the November 10, 2021 build of plugin-extras are:
1) AtomTablePlugin
2) AvgCpuPlugin
3) DbgViewPlugin
4) DnsCachePlugin
5) DpiAwarenessExtPlugin
6) ExtraCpuGraphsPlugin
7) GrowlNotifications
8) MemoryExtPlugin
9) NvGpuPlugin
10) PerfMonPlugin
11) ProductPolicyPlugin
12) ROTViewerPlugin
13) SecurityExplorer
14) ServiceBackupRestorePlugin
15) SetCriticalPlugin
16) TaskbarExtPlugin
17) TerminatorPlugin
18) TrustedInstallerPlugin

So some of these may already be imported in System Informer, I thought TerminatorPlugin for example would be one people would still need in Windows 10/11.

**Comment by MagicAndre1981** [comment]
@dmex what about the extra plugins? Will they ever be migrated? I would like to use the TaskbarExtPlugin again.

**Reply by dmex** [reply]
> what about the extra plugins? Will they ever be migrated?

Some yes, others no.

> I would like to use the TaskbarExtPlugin again.

TaskbarExtPlugin was already merged like 3 years ago? You've been able to configure it from System Informer > Options window > ToolStatus > Graph dropdown at the bottom:

![image](https://github.com/winsiderss/systeminformer/assets/1306177/d6b95b4e-0655-46f3-a9b3-1d32ce95a7b2)

**Reply by MagicAndre1981** [reply]
What? I totally overlooked this 🤦‍♂️. Maybe move it into an own entry or to General
